.github/workflows/package.json

```diff
@@ -7,8 +7,10 @@
     "@octokit/core": "^4.0.4",
     "@testmo/testmo-cli": "^1.4.0",
     "commander": "^9.4.0",
+    "fs": "^0.0.1-security",
```

Comment:
issue: If you need to use `fs` from Node.js, you don't need to add it as dependency, since it's native module. Adding this can be potentially dangerous if package is ever changed, for now npm has simply made this package empty: https://www.npmjs.com/package/fs

In case this was added because of some linter issue, we probably need to update it's settings.